fern/docs.yml

```
@@ -39,6 +39,8 @@ navigation:
         contents:
           - page: Knowledge
             path: ./docs/pages/train/building-knowledge-base.mdx
+          - page: Preparing knowledge
+            path: ./docs/pages/train/preparing-knowledge.mdx
```

```suggestion
            path: ./docs/pages/preparing-knowledge.mdx
```

i don't believe your page is in the train directory